src/main/scala/com/lightbend/lagom/docs/DocumentationGenerator.scala

```diff
@@ -81,8 +81,7 @@ object DocumentationGenerator extends App {
     "get-started-java.html" -> html.getstartedjava,
     "get-started-java-sbt.html" -> html.getstartedjavasbt,
     "get-started-java-maven.html" -> html.getstartedjavamaven,
-    "get-started-scala.html" -> html.getstartedscala,
-    "faq.html" ->html.faq
+    "get-started-scala.html" -> html.getstartedscala
```

Comment:
Isn't that removing the the FAQ from the top bar? If so, we should link to the FAQ from somewhere else.

Reply:
No, this just feeds the generator a list of Twirl templates to run. This is no longer a Twirl template, so it was removed. It still appears in the top bar, which is defined in `topNav.scala.html`.